src/app/dashboard/components/dashboard-under-follow-up/dashboard-under-follow-up.component.scss

```diff
@@ -1 +1,146 @@
-/* stylelint-disable */
+@use '@angular/material' as mat;
+@use '/src/variables.scss' as *;
+
+h2 {
+  font-size: 24px;
+  color: mat.get-color-from-palette($sormas-accent, 400);
+
+  strong {
+    font-weight: 500;
+    color: mat.get-color-from-palette($sormas-accent, 800);
+  }
+  .icon {
+    color: mat.get-color-from-palette($sormas-primary, 500);
+    vertical-align: middle;
+    font-size: 20px;
+  }
+}
+
+h3 {
+  font-size: 20px;
+  color: mat.get-color-from-palette($sormas-accent, 500);
+  text-align: center;
+  font-weight: 500;
+  margin-top: 30px;
+}
+
+.not-visited-day-texts {
+  margin: 5px 0;
+}
+
+.not-visited-days-count {
+  margin-bottom: 5px !important;
+}
+
+.not-visited-days {
+  text-align: center;
+  margin-left: 0;
+  margin-right: 0;
+  padding: 0;
+  list-style: none;
+  font-size: 14px;
+  font-weight: 500;
+  color: mat.get-color-from-palette($sormas-accent, 500);
+
+  .day {
+    line-height: 20px;
+    padding: 0 12px;
+    margin: 0 10px;
+  }
+
+  .text {
+    line-height: 20px;
+    padding: 0;
+    margin: 0 7px;
+    font-size: 12px;
+    font-weight: normal;
+  }
+
+  .day1 {
+    color: mat.get-color-from-palette($sormas-accent, 300);
+    text-align: center !important;
+    border-bottom: solid 2px mat.get-color-from-palette($sormas-accent, 300);
+  }
+  .day2 {
+    color: mat.get-color-from-palette($sormas-accent, 500);
+    text-align: center !important;
+    border-bottom: solid 2px mat.get-color-from-palette($sormas-accent, 500);
+  }
+  .day3 {
+    color: mat.get-color-from-palette($sormas-warning, 700);
+    text-align: center !important;
+    border-bottom: solid 2px mat.get-color-from-palette($sormas-warning, 700);
+  }
+  .day4 {
+    color: mat.get-color-from-palette($sormas-alert, 300);
+    text-align: center !important;
+    border-bottom: solid 2px mat.get-color-from-palette($sormas-alert, 300);
```

Comment:
can you please look for a solution so you do not use !important?

Reply:
Resolved. Didn't even needed for the current version. Thx!